demodoctrine/js/.webpack/common.js

```diff
@@ -0,0 +1,83 @@
+const fs = require('fs');
+const path = require('path');
+const webpack = require('webpack');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const cssExtractedFileName = 'theme';
+
+// PrestShop folders, we use process.env.PWD instead of __dirname in case the module is symlinked
+const psRootDir = path.resolve(process.env.PWD, '../../../');
```

Comment:
@NeOMakinG this part I'm less convinced, it requires the module to be in a shop `modules` folder to allow us to build assets, which makes sense when you are developing it

But it prevents us to build the module's assets independently, the thing is I don't see how we can get the core components as dependencies since we don't have a library for them..

Reply:
Aie, I think we can't do anything else, but you'r right this is touchy and a bad practice.

What about pushing these core dependencies inside an NPM module like inside a monorepo ? So every modules/themes can install core deps using NPM

@PierreRambaud  wdyt?